orttraining/orttraining/test/training_ops/cpu/nn/dropout_op_test.cc

```diff
@@ -109,6 +109,101 @@ void RunDropoutTest(const char* op, const bool use_mask, const std::vector<int64
 
   t.Run(OpTester::ExpectResult::kExpectSuccess, "", {}, nullptr, nullptr, ExecutionMode::ORT_SEQUENTIAL, output_verifier);
 }
+
+void RunBiasDropoutTest(const char* op, const bool use_mask, const std::vector<int64_t>& input_shape, float ratio = -1,
+                        bool training_mode = true, bool use_float16_ratio = false, bool has_residual = true) {
+  OpTester t{op, 1, kMSDomain};
+  const int64_t seed = 42;
+  t.AddAttribute("seed", seed);
+
+  const auto input_size = std::accumulate(
+      input_shape.begin(), input_shape.end(), static_cast<int64_t>(1), std::multiplies<>{});
+  std::vector<float> input(input_size);
```

Comment:
>input [](start = 21, length = 5)

consider using ValueRange()
[https://github.com/microsoft/onnxruntime/blob/12367a6b11e96e70112026cb7975f7035c7ad0ed/onnxruntime/test/common/tensor_op_test_utils.h#L86](https://github.com/microsoft/onnxruntime/blob/12367a6b11e96e70112026cb7975f7035c7ad0ed/onnxruntime/test/common/tensor_op_test_utils.h#L86)
then the vector can be const too